frontend/src/components/Stepper/components/Footer.js

```diff
@@ -0,0 +1,29 @@
+/*
+  This footer should be used with inside <Pager /> or <Stepper /> forms. I feel
+  like this class may be able to be refactored using 'render-props' for a better
+  developer experience when creating forms.
```

Comment:
I like this call-out of a potential future gain without getting bogged down today in something we might not need.